### DISCUSSION TITLE
does Visual Studio has some plugin for azsli shader

### DISCUSSION THREAD

**Discussion by zhangbaochong** [o3de/o3de#9993] [Q&A]
![image](https://user-images.githubusercontent.com/16303584/172750114-a61bd278-a8e1-416b-9c81-bc0dfed5ba6b.png)
I use VS open the source file, while it cannot include other azsli file，and also cannot jump between functions.
How to do ? any plugin for this ? or chang to another IDE?

**Comment by AMZN-alexpete** [comment]
I typically add the .azsli extension to VS and use the existing .hlsl syntax highlighting because it is so similar.  I also prefer to use "open folder" VS Code option for shader editing instead of opening the .sln file in Visual Studio because it seems to make jumping around to files and functions a little bit easier.
